suite-native/app/app.config.ts

```diff
@@ -117,56 +186,13 @@ export default ({ config }: ConfigContext): ExpoConfig => {
                 UIRequiredDeviceCapabilities: ['armv7'],
             },
         },
-        plugins: [
-            [
-                'expo-font',
-                {
-                    fonts: [
-                        '../../packages/theme/fonts/TTSatoshi-Medium.otf',
-                        '../../packages/theme/fonts/TTSatoshi-DemiBold.otf',
-                    ],
-                },
-            ],
-            [
-                '@sentry/react-native/expo',
-                {
-                    url: 'https://sentry.io/',
-                    project: 'suite-native',
-                    organization: 'satoshilabs',
-                },
-            ],
-            [
-                'expo-barcode-scanner',
-                {
-                    cameraPermission:
-                        'Allow $(PRODUCT_NAME) to access camera for QR code scanning.',
-                },
-            ],
-            [
-                'expo-build-properties',
-                {
-                    android: {
-                        minSdkVersion: 28,
-                    },
-                    ios: {
-                        deploymentTarget: '14.0',
-                        flipper: 'true',
-                    },
-                },
-            ],
-            '@trezor/react-native-usb/plugins/withUSBDevice.js',
-            // Define FLIPPER_VERSION
-            './plugins/withGradleProperties.js',
-            // These should come last
-            './plugins/withRemoveXcodeLocalEnv.js',
-            ['./plugins/withEnvFile.js', { buildType }],
-            './plugins/withRemoveiOSNotificationEntitlement.js',
-        ],
+        plugins: getPlugins(),
         extra: {
             commitHash: process.env.EAS_BUILD_GIT_COMMIT_HASH || '',
             eas: {
                 projectId,
             },
+            isDetoxBuild: !!process.env.IS_DETOX_BUILD,
```

Comment:
Is this necessary? I think you should be able to access `process.env.IS_DETOX_BUILD` without adding it to constants.

Reply:
You might just need to use `EXPO_PUBLIC_` prefix for expo to include it in the env variables.

Reply:
I need to "bake in" the information if the build contains detox logic already during the build process. How could I pass this information to the already builded app later? 

If there is some simpler way, let's do it.

Reply:
Would be best to implement it using environment variables passed from build. I guess it could somehow inject the value into `process.env.IS_DETOX_BUILD`. If not, try adding the `EXPO_PUBLIC_` prefix and hopefully then it becomes available in the code. If not, let's not waste time on this...